pkg/util/convert/convert.go

```diff
@@ -0,0 +1,87 @@
+package convert
+
+import (
+	"fmt"
+
+	"sigs.k8s.io/yaml"
+
+	kudov1beta1 "github.com/kudobuilder/kudo/pkg/apis/kudo/v1beta1"
+)
+
+// String returns a pointer to the string value passed in.
+func String(v string) *string {
```

Comment:
this might make more sense to call `StringPtr`

Reply:
`v` doesn't make sense to me either...  why `v`?

Reply:
This function is unchanged from its origin in `pkg/util/kudo`. Happy to rename it though. As for variable names, `v` seems to stand for "value". I'm not a fan of these short variable names but didn't chose to change it and kept using this convention in the new functions added in this file as this way of naming parameters seems to be common in our code base.